common/src/main/scala/com/aliyun/emr/rss/common/meta/PartitionLocationInfo.scala

```diff
@@ -301,7 +301,13 @@ class PartitionLocationInfo extends Logging {
     getAllIds(shuffleKey, slavePartitionLocations)
   }
 
-  def close(): Unit = {
-    logInfo("Start close " + this.getClass.getSimpleName)
+  def isEmpty: Boolean = masterPartitionLocations.isEmpty && slavePartitionLocations.isEmpty
```

Comment:
Shall we add synchronization here? Most methods in this class are sync.

Reply:
Done